controller/lib/core/blower_fsm.h

```diff
@@ -82,6 +82,11 @@ struct BlowerSystemState {
   // This is defaulted to false here and is handled by BlowerFsm, rather than
   // the individual FSM classes (OffFsm, PressureControlFsm, etc).
   bool is_new_breath = false;
+
+  // Is this the first BlowerSystemState for a state different from Off state?
+  // Used to reset some variables (volume integrators) to 0 when we start the
+  // ventilator (transition from Off mode to any other mode).
+  bool start_ventilation = false;
```

Comment:
Could the Controller instead detect start_ventilation by noticing a transition from blower_setpoint == nullopt to blower_setpoint != nullopt?

That seems a little simpler to me; honestly I am already a little confused about the new comment about is_new_breath.

Reply:
Yes, it is probably simpler, I just didn't notice it would be that easy!
I'll try to make the is_new_breath comment easier to follow as well.